src/sentry/issues/ongoing.py

```diff
@@ -39,6 +40,14 @@ def bulk_transition_group_to_ongoing(
     for group in groups_to_transistion:
         group.status = GroupStatus.UNRESOLVED
         group.substatus = GroupSubStatus.ONGOING
+        if from_status != GroupStatus.UNRESOLVED:
+            issue_unresolved.send_robust(
```

Comment:
@scefali I'm making user nullable in this analytics record because sometimes like this case there's no user. Let me know if that's an issue.

Reply:
@sentaur-athena it won't got to amplitude but that's fine